docs/Earthfile

```diff
@@ -1,16 +1,35 @@
 VERSION 0.7
 
-deps:
-    FROM python:3.11-bullseye
-    DO ../earthly/python+POETRY_SETUP
+# cspell: words runable
 
+# Copy all the source we need to build the docs
 src:
-    FROM +deps
+    FROM ../earthly/docs+mkdocs-material
 
-    COPY --dir src mkdocs.yml .
+    # Just copy the actual doc src here.
+    COPY --dir src includes macros overrides mkdocs.yml /docs
 
+    # Now copy into that any artifacts we pull from the builds.
+    COPY --dir ../+repo-docs/repo /docs/includes
+
+
+# Build the docs here.
```

Comment:
Nitpick: can we come to an agreement on the level of verbosity that we want to hit with comments? This target reads like a narrative, but I'm not sure it's really necessary. I try to save comments for tricky parts that are not obvious by just looking at it. 

You can see my other Earthfiles. Most of them have no comments except for edge cases that need clarity. So there's a pretty dramatic difference between our two styles :)

Reply:
My `comment` knob was stuck at `11` many years ago.  I fear if I try and force it back down I will just break it off instead.

What we could do, is leave them in for now,  I am going to be making a ticket for a dev to add the "how to use docs" part of the cat-ci guide documentation.  At that time, they can eliminate the comments I have here, and translate it into the guide page itself.

Reply:
Done